Oneirocom/Magick
# Debugging switch

## What Changed:

Added switch to every node which can be used to toggle on sending run information to the console.

## How to test:

Run a graph.  Nothing should be logging to the IDE console.  Select a node, toggle the 'log' switch on.  That node should log data to the console.
